service/src/main/java/bio/terra/externalcreds/dataAccess/LinkedAccountDAO.java

```diff
@@ -32,10 +34,23 @@ public LinkedAccountDAO(NamedParameterJdbcTemplate jdbcTemplate) {
             jdbcTemplate.query(query, namedParameters, new LinkedAccountRowMapper())));
   }
 
+  public List<LinkedAccount> getExpiringLinkedAccounts(Timestamp expirationCutoff) {
+    var namedParameters = new MapSqlParameterSource("expirationCutoff", expirationCutoff);
+    var query =
+        "SELECT DISTINCT la.* FROM linked_account la"
+            + " JOIN ga4gh_passport passport"
+            + " ON passport.linked_account_id = la.id"
+            + " LEFT JOIN ga4gh_visa visa"
+            + " ON visa.passport_id = passport.id"
+            + " WHERE passport.expires <= :expirationCutoff"
```

Comment:
Yes! Good point! In theory, any linked accounts with `la.is_authenticated = false` shouldn't have passports (and won't be returned by the query), but it's definitely better to check just in case.

This is fixed now.